src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -762,7 +776,12 @@ export default function CDPipeline({
         setFormData(_form)
     }
 
-    const validateStage = (stageName: string, _formData: PipelineFormType, formDataErrorObject?): void => {
+    const validateStage = (
+        stageName: string,
+        _formData: PipelineFormType,
+        formDataErrorObject?,
+        _clonedPluginDataStore?: typeof pluginDataStore,
```

Comment:
Not being used but need to add since in CIPipeline same signature is there and ESLint is asking to append _ to unused variables